libgalois/include/katana/PropertyGraph.h

```diff
@@ -762,6 +762,27 @@ class KATANA_EXPORT PropertyGraph {
   edge_indexes() const {
     return edge_indexes_;
   }
+
+  // Returns true of an index exists for the named property
```

Comment:
*if
obviously don't need to recommit for that lol

Reply:
Yeah, noticed that today - will *try* to keep in mind to fix it.